### ISSUE TITLE
Document Windows and macOS support

### ISSUE THREAD

**Issue by mrjones-plip** [medic/medic-os#52]
Right now it's assumed that our `docker-compose up` calls are made on Linux. We should add documentation on how to use Docker Desktop on both macOS and Windows to successfully run them well.

this may include updating the [localsetup](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/) and [developing on windows](https://docs.communityhealthtoolkit.org/core/guides/using-windows/).

**Comment by Hareet** [comment]
Hey @mrjones-plip - We had the intention of formally supporting these operating systems (production environment) when we first spoke. I think lets drop that level of support to unblock your tasks, we are usually down an SRE recently and I'm afraid of scope increasing as unknowns arise or more workarounds needed for various Windows installs. 

Are you okay with making this ticket solely related to the dev environment so app-services/dev-team can work out the workaround scripts and documentation without SRE? A note that says production runs Ubuntu 16.04 and that is our only currently supported operating system till arch v3 is released.

**Comment by mrjones-plip** [comment]
This is correctly prioritized as the last in the queue and, yes, didn't think that it would apply to production, dev only.  It is both OK to wait on this ticket and to focus on dev only fix.

My two cents are that I don't think we should plan to support Windows or macOS for production hosting at all!  But this is not my call to make ;)

I'll reach out on Slack about the "to unblock your tasks" comment to ensure we agree on which team will be tackling [the tasks](https://github.com/medic/cht-core/issues/7061#issuecomment-832190318).

**Comment by mrjones-plip** [comment]
Closing as "won't fix" in hopes that we've done is enough.  Will reach out to app services/Binod about improvements to help script as it stands today, this documentation aside.
